package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "virtual-keyboard",
+  "version": "1.0.0",
+  "description": "Virtual keyboard",
+  "main": "index.js",
+  "scripts": {
+    "lint": "npx eslint .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Crepiks/virtual-keyboard.git"
+  },
+  "author": "Sayazhan Onglassyn aka Crepiks",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Crepiks/virtual-keyboard/issues"
+  },
+  "homepage": "https://github.com/Crepiks/virtual-keyboard#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-react": "^7.19.0",
+    "eslint-plugin-react-hooks": "^2.5.0",
```

Comment:
I'm not sure where you've found this, but plugin for react is not needed at all.

Reply:
Done

Reply:
I have used incorrect config. I had to use eslint-airbnb-config-base instead eslint-airbnb-config, which is made for reactjs projects